src/ngraph_utils.cc

```diff
@@ -362,8 +361,7 @@ std::string GraphFilenamePrefix(std::string kind, int idx, int sub_idx) {
   ss << GraphFilenamePrefix(kind, idx) << "_" << std::setfill('0')
      << std::setw(4) << sub_idx;
 #if defined NGRAPH_DISTRIBUTED
-  ngraph::Distributed dist;
-  int Rank_ID = dist.get_rank();
+  int Rank_ID = ng::get_distributed_interface()->get_rank();
```

Comment:
Why Rank_ID is using unconventional naming scheme? Since this is a local variable - chance this to rank_id or something similar.